docs/howto/interop/nholthaus.md

```diff
@@ -0,0 +1,70 @@
+# `nholthaus/units` Interoperation
+
+The Au repository includes ready-made conversion machinery for the
+[nholthaus/units](https://github.com/nholthaus/units) library.  If you're migrating to or from this
+library, you don't need to use our [more general guide](./index.md).  Instead, you can follow this
+guide, which is both more specific and less labor-intensive.
+
+Once you're finished with this guide, you'll be able to pass any nholthaus type to an API expecting
+an equivalent Au quantity type, and vice versa!
+
+## Step 1: Add the file to your project
+
+The goal of this step is to make sure the file with the conversion machinery is available in your
+project, and to know its include path.
+
+This will depend on which [installation method](../../install.md) you used: single-file, or full
+install.
+
+- **Full Install:**  In this case, the file is already available.  Assuming `au/` is the root of
+  your Au includes, the include path will be `"au/compatibility/nholthaus_units.hh"`.
+
+- **Single-file Install:**  Copy the file `compatibility/nholthaus_units.hh` from the Au repository
+  into a suitable location in your project.  Make a note of the include path.
+
+## Step 2: Create a shim
+
+The file we provide, `compatibility/nholthaus_units.hh`, is written in a particular and unusual way:
+it doesn't have any explicit dependencies on either Au or nholthaus units.  This is because we can't
+know how you've included either library in your project.
+
+One consequence of this is that _include order matters_: this file must be included **after** both
+the Au and nholthaus libraries.  In general, include order dependence is very fragile and should be
+avoided.  _However_, it can be done safely if we confine this dependence to a single file --- the
+_shim_ --- and provide _only that file_ to our end users.  This shim must do the following three
+things, in this order.
+
+1. Include the nholthaus-relevant parts of Au (in whatever manner is appropriate for your project).
+2. Include the nholthaus units library (in whatever manner is appropriate for your project).
+3. Include your copy of `compatibility/nholthaus_units.hh`, **after** every other file.
+    - (The include path comes from Step 1 above.)
+
+We have included a [self-explaining example
```

Comment:
"Include" seems to have shifted in meaning between the previous paragraph/steps and _this_ paragraph -- is that fair? If that's true, maybe we shift "include" to "provide" in this paragraph.

Reply:
Great catch!  Done.